redwood-protocol-host/src/commonMain/kotlin/app/cash/redwood/protocol/host/HostProtocolAdapter.kt

```diff
@@ -425,7 +425,7 @@ private class RootProtocolNode<W : Any>(
     }
 
   override fun detach() {
-    children.detach()
+    // Do nothing because 'children' is owned by the host's TreehouseView.
```

Comment:
The Redwood protocol doesn't _strictly_ have to be used by Treehouse, although that is certainly the overwhelmingly common usage. Maybe put a note on the class docs up above, or on `close()`.

Reply:
Oh even better. I thought it was something specific to treehouse.